web/src/setupTests.js

```diff
@@ -1,4 +0,0 @@
-import { configure } from 'enzyme';
```

Comment:
removed a couple of `setupTests` files that didn't look like they're being used rather than try to fix the formatting issues in them